ethernet-view/src/pages/LoggerPage/LoggerPage.tsx

```diff
@@ -1,18 +1,16 @@
-import { Orientation } from "hooks/useSplit/Orientation";
-import { SplitLayout } from "layouts/SplitLayout/SplitLayout";
 import { ChartsColumn } from "pages/LoggerPage/ChartsColumn/ChartsColumn";
-import chart from "assets/svg/chart.svg"
+import { LogsColumn } from "./LogsColumn/LogsColumn";
+import styles from "./LoggerPage.module.scss";
 
 export const LoggerPage = () => {
     return (
-        <SplitLayout
-            components={[
-                {
-                    component: <ChartsColumn />,
-                    collapsedIcon: chart,
-                },
-            ]}
-            orientation={Orientation.HORIZONTAL}
-        ></SplitLayout>
+        <div className={styles.LoggerPage}>
+            <div className={styles.LogsColumn}>
+                <LogsColumn />
+            </div>
+            <div className={styles.ChartsColumn}>
+                <ChartsColumn />
+            </div>
+        </div>
```

Comment:
I would keep the SplitLayout here so the user can hide the chart selector and make it bigger if the file name is too long or somehting

Reply:
That was my idea, but I would like to set an initial size for each component to make the LogLoader set to 30% and Charts to 70%, maybe, but now it is not possible to do it. I will try to implement it in this PR.

Reply:
Ok